impls/vbs/env.vbs

```diff
@@ -3,61 +3,36 @@ Option Explicit
 Function NewEnv(objOuter)
 	Dim varRet
 	Set varRet = New Environment
-	Set varRet.Self = varRet
```

Comment:
Seems to be the only way to get the object itself in VBS.

Reply:
Well, `Me` keyword exist (although it's not mentioned in the manual), the problem should not be here.